test/test_manifest.json

```diff
@@ -2080,5 +2080,11 @@
        "md5": "d5b37f8bf1b3aafa1b4fcf19ebdc7c74",
        "rounds": 1,
        "type": "eq"
+    },
+    {  "id": "issue5909",
+       "file": "pdfs/issue5909.pdf",
+       "md5": "65c169b6f540b27ac0ff2738a80d1e14",
```

Comment:
@jordan-thoms Perhaps you forgot to update the MD5 hash after updating the test file?